cmd/rollouts-controller/main.go

```diff
@@ -48,6 +48,8 @@ func newCommand() *cobra.Command {
 		metricsPort          int
 		healthzPort          int
 		instanceID           string
+		qps                  float32
```

Comment:
Does it make sense to have QPS also as an `int` instead of a `float`?

Reply:
I think this is to make it consistent with the client-go type: https://github.com/kubernetes/client-go/blob/d85d0d914a32b2cd96ebfd5f65f2087d6272f70c/rest/config.go#L45